prime-react/src/main/scala/lucuma/react/primereact/Checkbox.scala

```diff
@@ -29,14 +32,25 @@ case class Checkbox(
 
 object Checkbox {
   private val component = ScalaFnComponent[Checkbox] { props =>
+    val changeHandler: (CheckboxChangeEvent, Boolean) => Callback =
+      (e, b) =>
+        props.onChange.toOption
+          .map(_(b))
+          .orElse(props.onChangeE.toOption.map(_(b, e.originalEvent.get)))
+          .getOrElse(Callback.empty)
+
     CCheckbox(props.checked)
       .applyOrNot(props.id, _.id(_))
       .applyOrNot(props.inputId, _.inputId(_))
       .applyOrNot(props.disabled, _.disabled(_))
       .applyOrNot(props.clazz, (c, p) => c.className(p.htmlClass))
       .applyOrNot(props.tooltip, _.tooltip(_))
       .applyOrNot(props.tooltipOptions, (c, p) => c.tooltipOptions(p.asInstanceOf[CTooltipOptions]))
-      .applyOrNot(props.onChange, (c, p) => c.onChange(iwcp => p(iwcp.checked.getOrElse(false))))(
+      .applyOrNot(
```

Comment:
You should be able to support both `onChange` and `onChangeE` handlers at the same time and then call `.onChange` here rather than `.applyOrNot`. We do that for Button, MultiSelect and others. MultiSelect is probably a good example. https://github.com/gemini-hlsw/lucuma-react/blob/31360308791a9c7ee3c6ad03f22f4971ef368f37/prime-react/src/main/scala/lucuma/react/primereact/MultiSelect.scala#L146